tests/utils.py

```diff
@@ -17,8 +16,6 @@ class InputIdentifiers(str, Enum):
     predictions = _PREDICTIONS_IDENTIFIER
     opener = "opener"
     datasamples = "datasamples"
```

Comment:
The identifiers that are fixed in substra-tools should not be in this utils enum `InputIdentifiers`, but use directly `TASK_IO_DATASAMPLES` and such

Reply:
done